src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import "./App.css";
+import { useState } from "react";
+import { useEffect } from "react";
+import divider from "./assets/divider.svg";
+import dice from "./assets/dice.svg";
+
+function App() {
+  const [adviceNumber, setAdviceNumber] = useState(null);
+  const [adviceText, setAdviceText] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const getAdvice = async () => {
+    try {
+      setIsLoading(true);
+
+      const response = await fetch("https://api.adviceslip.com/advice");
+      const data = await response.json();
+      const slip = data.slip;
+      const number = slip.id;
+      const text = slip.advice;
+
+      setAdviceNumber(number);
+      setAdviceText(text);
+    } catch (error) {
+      console.log(error);
+    } finally {
+      setIsLoading(false);
```

Comment:
`setTimeout` is not necessary for it to work, but it makes the animation look a bit better 

```suggestion
     setTimeout(() => {
          setIsLoading(false);
     }, 700);
```